src/components/LeftDrawer/LeftDrawer.tsx

```
@@ -1,74 +1,159 @@
+  // Render a drawer item with icon
+  const renderDrawerItem = useCallback(
+    (to: string, icon: React.ReactNode, label: string, testId: string) => (
+      <NavLink key={to} to={to} onClick={handleLinkClick}>
+        {({ isActive }) => (
+          <button
+            className={`${
+              isActive ? styles.sidebarBtnActive : styles.sidebarBtn
+            }`}
+            data-testid={testId}
+          >
+            <div className={styles.iconWrapper}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)